app/helpers/search_helpers.py

```diff
@@ -1,6 +1,18 @@
 from math import ceil
 
 
+def get_lot_label(lot_param_value):
```

Comment:
<sub>There's also [this function](https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/master/app/helpers/service_helpers.py) that seems to do the same thing.</sub>